src/models/findById.ts

```diff
@@ -0,0 +1,10 @@
+import users from '../database/users';
+import { User } from '../types/User';
+
+const findById = (id: string): Promise<User> => {
```

Comment:
I think you should check if the id is valid uuid and if not response that id is not valid instead of not found